src/verify/VerificationEducationScreen.tsx

```diff
@@ -137,6 +138,13 @@ function VerificationEducationScreenDecentralized({ route, navigation }: Props)
     }
   }
 
+  // Set showSkipDialog to false during component unmount
+  useEffect(() => {
+    return () => {
+      navigation.setParams({ showSkipDialog: false })
```

Comment:
ah, this causes problems with connecting phone number from the settings screen since we rely on the `hideOnboardingStep` navigation param to be persisted during navigation to the country selection screen (watch how the header changes after the country screen is dismissed) :( i don't think this fixes the problem either, for me the skip dialog continues to block the app. can we remove this change?

https://user-images.githubusercontent.com/20150449/201121005-8f009f67-36a0-4738-9701-19a0c1cc4a9e.mp4